README.md

```diff
@@ -78,9 +78,10 @@ specifications as appropriate.
 backport version of `libseccomp-dev` is required. See [travis.yml](.travis.yml) for an example on trusty.
 * **btrfs development library.** Required by containerd btrfs support. `btrfs-tools`(Ubuntu, Debian) / `btrfs-progs-devel`(Fedora, CentOS, RHEL)
 2. Install **`socat`** (required by portforward).
-2. Install and setup a go 1.13.9 development environment. (Note: You can check the travis logs for a recent pull request to confirm the version(s) of golang currently being used to build and test master.)
-3. Make a local clone of this repository.
-4. Install binary dependencies by running the following command from your cloned `cri/` project directory:
+3. Install **`pkg-config`** (required by ??).
```

Comment:
not sure why you had to install that one...

Reply:
It would seem that is is need when linking the `libseccomp` library during the install of dependencies.

```console
$ make install.deps
.install.deps.linux
Cloning into '/tmp/cri-install-runc.ZSri/src/github.com/opencontainers/runc'...
[...]

# pkg-config --cflags  -- libseccomp libseccomp
pkg-config: exec: "pkg-config": executable file not found in $PATH
Makefile:44: recipe for target 'static' failed
make[1]: *** [static] Error 2
make[1]: Leaving directory '/tmp/cri-install-runc.ZSri/src/github.com/opencontainers/runc'
Makefile:170: recipe for target '.install.deps.linux' failed
make: *** [.install.deps.linux] Error 2
```